### PR TITLE
chore(demo): update `Textfield` + `Maskito` example

### LINKED ISSUE #9679: 🐞 - value in the input is overlapped by a `tuiTextfieldPrefix` when scrolling
### Reproduction url

https://stackblitz.com/edit/angular-fhtqky

### Description

If a value in an input that uses a prefix is long, the value in the input is overlapped by a `tuiTextfieldPrefix` when scrolling.

Image

### Taiga UI version

3.89.0

### Browser Used

Chrome

### OS Used

macOS

### PR DESCRIPTION
**Pull request by nsbarsukov**
Fixes #9679

### CODE REVIEW

File: projects/demo/src/modules/components/textfield/examples/5/index.html [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+        #input
         placeholder="3,14"
         tuiTextfield
         [maskito]="options"
         [(ngModel)]="value"
     />
+
+    <button
+        *ngIf="value"
+        appearance="icon"
```
Comment (waterplea) [inline projects/demo/src/modules/components/textfield/examples/5/index.html:14]: Note to self: see if textfield should set icon as button default appearance, and xs as size

**Review by waterplea — APPROVED** [review]

**Comment by waterplea** [comment]
Custom cleaner doesn't seem to work

**Comment by nsbarsukov** [comment]
> Custom cleaner doesn't seem to work

🤔 

https://github.com/user-attachments/assets/4ec34212-43c6-4952-bdef-3a5cbf312294

File: package.json [hunk 2]
```diff
@@ -86,7 +86,8 @@
             "i18n/no-japanese-character": "off",
             "i18n/no-thai-character": "off",
             "jest/prefer-importing-jest-globals": "off",
-            "sonarjs/prefer-nullish-coalescing": "off"
+            "sonarjs/prefer-nullish-coalescing": "off",
+            "@angular-eslint/template/elements-content": "off"
```
[the remark this answers is not in the document]
Comment (nsbarsukov) [reply package.json:90]:
```html
<button
    *ngIf="value"
    appearance="icon"
    iconStart="@tui.x"
    size="xs"
    tabindex="-1"
    tuiIconButton
    type="button"
    (click)="clear()"
    (pointerdown.silent.prevent)="input.focus()"
></button>
```

```
eslint projects/demo/src/modules/components/textfield/examples/5/index.html
| Processing: projects/demo/src/modules/components/textfield/examples/5/index.html 

/Users/n.barsukov/WebstormProjects/taiga-ui/projects/demo/src/modules/components/textfield/examples/5/index.html
  12:5  error  <button> should have content  @angular-eslint/template/elements-content

✖ 1 problem (1 error, 0 warnings)
```

[the remark this answers is not in the document]
Comment (splincode) [reply package.json:90]: https://github.com/taiga-family/taiga-ui/pull/9449

you can add `title` attribute for button

[the remark this answers is not in the document]
Comment (waterplea) [reply package.json:90]: It's an icon button, you can add "Clear" text inside it

[the remark this answers is not in the document]
Comment (nsbarsukov) [reply package.json]: Fixed

**Review by splincode — APPROVED** [review]

**Pull request merged**